packages/storybook/src/documentation/developer/community-component-aanleveren.md

```diff
@@ -0,0 +1,285 @@
+# Community Component Aanleveren
+
+[github-button]: https://github.com/nl-design-system/lux/tree/main/packages/storybook/src/web-components/button/button.mdx#L13
+[lux-button]: /docs/web-components-button--docs
+[mdn-slots]: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/slot
+[nlds-components]: https://www.nldesignsystem.nl/componenten/
+[nlds-estafettemodel]: https://www.nldesignsystem.nl/handboek/estafettemodel
+[nlds-utrecht-testing]: https://nl-design-system.github.io/utrecht/storybook-react/index.html?path=/docs/react-testing--docs
+[stencil]: https://stenciljs.com/docs/introduction
+[stencil-slots-doc]: https://stenciljs.com/docs/docs-json#slots
+[tsdoc]: https://tsdoc.org/
+
+Als gebruiker van LUX mag je ook componenten aandragen. Mocht het component al beschikbaar zijn in de [NL Design System community][nlds-components] is deze geschikt om bij LUX te betrekken. Dat kunnen wij voor je doen maar dat kan je zelf ook. Dan hoef je niet op ons te wachten. Handig toch?!
+
+Natuurlijk willen wij wel dat alle componenten op de zelfde wijze gebouwd worden zodat wij het allemaal makkelijker kunnen onderhouden. Om consistentie te waarborgen vertellen wij hier hoe je zelf een NLDS community component kunt aandragen als LUX component. Hier zijn de stappen.
+
+> **Waarom een wrapper?**
+>
+> Waarom gebruiken we niet direct het component vanuit de community? Is een LUX component nog wel nodig om het NLDS community component te gebruiken? Kan ik het niet gewoon zelf maken?
```

Comment:
Hier nog een kort antwoord op geven.

- de juiste token settings.
- output naar verschillende frameworks (React, Angular, Vue (evt)).
- wordt de community component geupdate, fixen we dat in de wrapper.
- wordt de community component `candidate` of `hall of fame`, updaten we dat in de wrapper.

Reply:
Juist, die moest ik nog doen

Reply:
Ik zit te denken om dit op een aparte pagina te beschrijven. Het is niet (duidelijk) in 2 of 3 zinnen uit te leggen en die content past niet bij de rest van de content op deze pagina. Wat vind jij? 

Dan laten we het voor nu even weg en zetten we dit terug als we die pagina hebben geschreven. En dan meteen daarheen linken, natuurlijk

Reply:
LUX-367 aangemaakt om die pagina te schrijven